frontend/src/components/Shared/ForgetPassword/forgetpassword.jsx

```
@@ -14,24 +54,40 @@
+          .split('; ')
+          .find(row => row.startsWith('resetCooldown='));
         
-        if (timeLeft > 0) {
-          setEmail(decodeURIComponent(email));
-          setCooldown(timeLeft);
-          startCooldownTimer(timeLeft);
-        } else {
-          document.cookie = 'resetCooldown=; expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/;';
+        if (cooldownData) {
+          const [email, expiry] = cooldownData.split('=')[1].split(':');
+          const timeLeft = Math.ceil((parseInt(expiry) - Date.now()) / 1000);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)